ob-pattern-1/templates/obam/configMaps/wso2ob-pattern-1-obam-bin.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wso2ob-pattern-1-obam-bin
+  namespace : {{ .Release.Namespace }}
+data:
+  {{- $file := .Files }}
+  {{- range $path, $byte := .Files.Glob "confs/obam/bin/*" }}
+  {{- $list := $path | splitList "/"}}
+  {{- $length := len $list }}
+  {{- $last := add $length -1 }}
+  {{ index $list $last }}: |-
+    {{- range $file.Lines $path }}
+      {{ . }}
+    {{- end }}
+  {{- end }}
+  
```

Comment:
@kdanW can you please check the suggested EOF line?

Applies to other instances as well.

Reply:
Is there any missing EOF lines in this file? I found missing EOF lines in 2 more files, but is anything wrong with this particular file?

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/a6c289bdb04eb294570ae486704103047497e89d